src/rpc/blockchain.cpp

```diff
@@ -527,15 +527,16 @@ UniValue getblock(const JSONRPCRequest& request)
 {
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
         throw std::runtime_error(
-            "getblock \"blockhash\" ( verbose )\n"
-            "\nIf verbose is false, returns a string that is serialized, hex-encoded data for block 'hash'.\n"
-            "If verbose is true, returns an Object with information about block <hash>.\n"
+            "getblock \"blockhash\" ( verbosity )\n"
+            "\nIf verbosity is 0, returns a string that is serialized, hex-encoded data for block 'hash'.\n"
+            "If verbosity is 1, returns an Object with information about block <hash>.\n"
+            "If verbosity is 2, returns an Object with information about block <hash> and information about each transaction. \n"
 
             "\nArguments:\n"
-            "1. \"blockhash\"     (string, required) The block hash\n"
-            "2. verbose           (boolean, optional, default=true) True for a json object, false for the hex encoded data\n"
+            "1. \"blockhash\"            (string, required) The block hash\n"
+            "2. verbosity              (numeric, optional, default=1) 0 for hex encoded data, 1 for a json object, and 2 for json object with transaction data\n"
 
-            "\nResult (for verbose = true):\n"
+            "\nResult (for verbosity = 1):\n"
```

Comment:
Shouldn't `Result (for verbose=false)` be changed and moved up to before `Result (for verbosity = 1)`?
```
"\nResult (for verbosity = 0):\n"
"\"data\"             (string) A string that is serialized, hex-encoded data for block 'hash'.\n"
```
(source: https://github.com/bitcoin/bitcoin/pull/8704/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0bR696-R697)

Shouldn't `Result (for verbosity = 2)` be added?
```
"\nResult (for verbosity = 2):\n"
"{\n"
"  ...,                     Same output as verbosity = 1.\n"
"  \"tx\" : [               (array of Objects) The transactions in the format of the getrawtransaction RPC. Different from verbosity = 1 \"tx\" result.\n"
"         ,...\n"
"  ],\n"
"  ,...                     Same output as verbosity = 1.\n"
"}\n"
```
(source: https://github.com/bitcoin/bitcoin/pull/8704/files#diff-decae4be02fb8a47ab4557fe74a9cb853bdfa3ec0fa1b515c0a1e5de91f4ad0bR722-R729)